notebooks/00-release-28.0.0/03-lookup-tables-28.ipynb

```diff
@@ -0,0 +1,1129 @@
+    "SELECT\n",
+    "  TIME_PARSE(\"depaturetime\") AS \"__time\",\n",
+    "  \"Reporting_Airline\",\n",
+    "  \"Tail_Number\",\n",
+    "  \"Distance\",\n",
+    "  \"Origin\",\n",
+    "  \"Dest\"\n",
+    "FROM \"ext\"\n",
+    "PARTITIONED BY DAY\n",
+    "'''\n",
+    "\n",
+    "display.run_task(sql)\n",
+    "sql_client.wait_until_ready('example-flights-lookup')\n",
+    "display.table('example-flights-lookup')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c3d6b39-6551-4b2a-bdfb-9606aa92c853",
+   "metadata": {},
+   "source": [
+    "### Import additional modules\n",
+    "\n",
+    "Run the following cell to import additional modules for this notebook.\n",
+    "\n",
+    "The cell also contains two functions - `postLookup` and `waitForLookup`, wrapping API posts to Druid when working with lookups.\n",
```

Comment:
@sergioferragut should these go into the wrapper?

Reply:
This would be a cool idea... I've logged as an Issue so we can come back to it.

Reply:
https://github.com/implydata/learn-druid/issues/49